src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -69,12 +69,15 @@
         public static final double RIGHT_WRAP = 60, RIGHT_WRAP_THRESHOLD = 90;
 
         // Current limits
-        public static final SupplyCurrentLimitConfiguration flywheelCurrentLimit = new SupplyCurrentLimitConfiguration(
+        public static final SupplyCurrentLimitConfiguration FLYWHEEL_CURRENT_LIMIT = new SupplyCurrentLimitConfiguration(
                 true, 40, 40, 500);
-        public static final SupplyCurrentLimitConfiguration turretCurrentLimit = new SupplyCurrentLimitConfiguration(
+        public static final SupplyCurrentLimitConfiguration TURRET_CURRENT_LIMIT = new SupplyCurrentLimitConfiguration(
                 true, 10, 10, 500);
-        public static final InterpolatingTreeMap dataPoints = InterpolatingTreeMap
+        public static final InterpolatingTreeMap DATA_POINTS = InterpolatingTreeMap
                 .fromCSV(new File(Filesystem.getDeployDirectory(), "shooterData.csv").getPath());
+
+        public static final double ALLOWED_FLYWHEEL_ERROR = 50;
+        public static final double ALLOWED_HOOD_ERROR = 1;
```

Comment:
1. `ALLOWED_FLYWHEEL_ERROR` is in velocity units (ticks/100ms), should we add a conversion from RPM to velocity?
2. (Mild nit) Please either move these two constants to the appropriate sections and rename to match `TURRET_ANGLE_TOLERANCE` or move `TURRET_ANGLE_TOLERANCE` here and rename to match these constants' names.